### PR TITLE
Adds missing Flutter criteria

### PR DESCRIPTION
**Pull request by danielroek**

### CODE REVIEW

File: Flutter/en/1.3.3.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Success Criterion 1.3.3 - Sensory Characteristics
+
+With Flutter, it is possible to use the [Semantics](https://api.flutter.dev/flutter/widgets/Semantics-class.html) widget to add more information to certain widgets in your application. To provide users with information about context actions the `hint` or `attributedHint` values can be used to describe what the result of the action will be. 
+
+To implement specific functionality for assistive technologies it is possible to add a `onTap`, `onLongPress` or other callbacks to the `Semantics` widget. In this case it is important to make sure the child nodes do not implement a touch listener, or to use  `excludeSemantics` to ignore these with the assistive technologies.
```
Comment (JJdeGroot) [inline Flutter/en/1.3.3.md]: double space between use and excludeSemantics

File: Flutter/en/1.4.11.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Success Criterion 1.4.11 - Non-text Contrast
+
+When designing an app, take the contrast ratio of all content into account. The contrast ratio should be at least 3:1.
+
+You can take a screenshot of the app to determine colors and calculate contrast.
```
Comment (JJdeGroot) [inline Flutter/en/1.4.11.md]: Are there any plugins for Flutter or IDE to check contrast?

File: Flutter/en/1.4.3.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Success Criterion 1.4.3 - Contrast (Minimum)
+
+When designing an app, take the contrast ratio of all text into account. The contrast ratio of text should be at least 4,5:1, or 3:1 for large text.
+
+You can take a screenshot of the app to determine colors and calculate contrast.
```
Comment (JJdeGroot) [inline Flutter/en/1.4.3.md]: Also add info about packages / IDE plugins if there are any added in 1.4.11

File: Flutter/en/2.4.1.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Success Criterion 2.4.1 - Bypass Blocks
+
+See the iOS and Android descriptions of this subject, as it is platform specific.
```
Comment (JJdeGroot) [inline Flutter/en/2.4.1.md]: Can you add internal links for iOS and Android, e.g. [iOS](../../ios/2.4.1.md) --> URL might be slightly different

File: Flutter/en/2.5.1.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Success Criterion 2.5.1 - Pointer Gestures
+
+With Flutter, it is possible to use a [GestureDetector]() to implement actions based on certain gestures made.
```
Comment (JJdeGroot) [inline Flutter/en/2.5.1.md]: Missing link

File: Flutter/en/1.4.11.md (hunk 2 above)
Comment (danielroek) [reply Flutter/en/1.4.11.md]: Not really, as Flutter does not build UI components in the same way as iOS or Android. I'd reccommend using some contrast calculator found on the web.

File: Flutter/en/1.4.3.md (hunk 3 above)
Comment (danielroek) [reply Flutter/en/1.4.3.md]: Added the advice to use a online contrast calculator

Comment (danielroek) [reply Flutter/en/1.4.3.md]: Let me know if you think this needs some more information.

**Pull request merged**